vm-virtio/src/mem.rs

```diff
@@ -695,7 +695,10 @@ impl MemEpollHandler {
                             if let Err(e) = &r {
                                 error!("{:?}", e);
```

Comment:
This error message should also be improved too.

Reply:
Fixed.